### DISCUSSION TITLE
why 'var' keyword removed in https://ziglearn.org chapters?

### DISCUSSION THREAD

**Discussion by veryverybadbadger** [Sobeston/zig.guide#146] [Q&A] [answered]
All pages in https://ziglearn.org/chapter-[0-5], every 'var' words removed but https://github.com/Sobeston/ziglearn/blob/master/chapter-1.md has them.

**Comment by veryverybadbadger** [comment]
hmm.. but safari shows 'var'. my chrome browser doesn't shows me 'var's in https://ziglearn.org/chapter-[0-5]

**Comment by veryverybadbadger** [comment]
![image](https://user-images.githubusercontent.com/5780066/165884081-88b76a32-c9ad-4a87-ae74-7fbbdad4aad4.png) in my chrome browser.

**Comment by veryverybadbadger** [comment]
![image](https://user-images.githubusercontent.com/5780066/165884321-cd0c1082-e117-4403-94b9-26ae201fec2e.png)
safari

**Comment by kristoff-it** [comment, marked as answer]
I just tried on Chrome (Windows) and it did not remove `var`s, here's a guess: overzealous adblock is the curlpit.

**Reply by ikskuh** [reply]
Can confirm with Chrome (Linux) and Firefox (Linux)
